interactivereduction/frontend
# Add reduction history page to display runs

Closes #33.

Can now view runs via the reduction history page.

## How to test

Get a populated database connected to the API. Run the API with `uvicorn ir_api.ir_api:app --host 0.0.0.0 --port 8080` on the `implement_specification_pattern` branch. Launch the plugin and see if `ir/reduction-history/{instrument}` is populated with runs for each instrument.

Build the frontend repo and test when ran through SciGateway.
